CONTRIBUTING.md

```diff
@@ -42,10 +42,10 @@ Projects that depend heavily on this client that should be taken into considerat
 
 ## Releasing
 
-```console
-$ npm version [major|minor|patch]
-$ npm publish
-```
+1. Merge the release PR
+2. Switch to the default branch `git checkout master`
+3. Pull latest changes `git pull`
+4. Publish the package `npm publish`
```

Comment:
At some point, we might want to have a common `CONTRIBUTING.md` to avoid duplication between repositories, what do you think?
At least for part of it.

Reply:
I agree, I forgot to do it in `node-template`. See https://github.com/netlify/node-template/pull/13

Reply:
I am also thinking of just putting some URL to point a common file, so we don't need to update all `CONTRIBUTING.md` when we make changes, what do you think?

Reply:
We could create a dedicate repo just for code of conduct and contributing guide or put it under https://github.com/netlify/eslint-config-node, WDYT?

Reply:
Either works! :+1: